src/server/controllers/logout.controller.js

```diff
@@ -0,0 +1,8 @@
+const serverConf = require('../conf/server');
+
+module.exports = (app) => {
+    app.get('/logout', (req, res) => {
+        res.clearCookie(serverConf.sessionCookieName);
```

Comment:
I beleive you should use passport method `logout` instead http://passportjs.org/docs/logout
And cause this logic is connected with auth, maybe it makes sence to move it to `src/server/controllers/auth.controller.js`?